lib/api/1.1/northbound/nodes.js

```diff
@@ -540,12 +546,14 @@ function nodesRouterFactory (
     router.get('/nodes/:identifier/workflows/active', rest(function (req) {
         return waterline.nodes.needByIdentifier(req.params.identifier)
         .then(function (node) {
-            return taskGraphProtocol.getActiveTaskGraph( { target: node.id })
-            .then(function (graph) {
-                if (graph) {
-                    return waterline.graphobjects.findOne({ instanceId: graph.instanceId });
-                }
-            });
+            return workflowApiService.findActiveGraphForTarget(node.id);
```

Comment:
I think we may want to consider that as follow-on work while we're pressing out more of the V2 API rather than adding lots of extensions under the 1.1 umbrella - @brianparry thoughts there? I normally balk at redundant routes, but in this case I think it would makes sense - either now (in the 1.1 API) or as an additional item under the v2.0 API set.